plugins/kafka/CMakeLists.txt

```diff
@@ -25,3 +26,5 @@ endif ()
 
 dependency_summary("rdkafka" RdKafka::rdkafka++ "Dependencies")
 target_link_libraries(kafka PUBLIC RdKafka::rdkafka++)
+
```

Comment:
Should there be a `dependency_summary` here?

Reply:
I think it would be redundant, because the AWSSDK is imported in multiple locations.